docs/getting-started.mdx

```diff
@@ -30,6 +22,11 @@ Decide what capability you would like to use in your App. A capability is descri
 
 In this guide we will use the capability **List user repositories of a VCS** like GitHub or GitLab. Its ID is `vcs/user-repos`.
 
+:::note Capability catalog
+You can search for other capabilities in [Superface catalog](https://superface.ai/catalog).
```

Comment:
@SuperVrata  We are kind of inconsistent in terminology here. Once you click on the [Superface catalog](https://superface.ai/catalog) there is no mention of `capability`, instead `use case` is used everywhere.

Reply:
I propose to replace `capability` with `use case` in documentation as well.

Reply:
`capability` has been long deprecated AFAIK

Reply:
`capability` is used all over our docs. I will open separate PR to fix it.